tests/e2e/settings-persistence-api.py

```
@@ -0,0 +1,63 @@
+Backend API test for settings persistence.
+Verifies that PUT /api/settings values are returned by GET /api/settings.
+Usage: python3 tests/e2e/settings-persistence-api.py [BASE_URL]
+"""
+import sys
+import requests
+
+BASE_URL = sys.argv[1] if len(sys.argv) > 1 else "http://localhost:3000"
+
+# 1. Get defaults
+print("[1/4] Fetching default settings...")
+resp = requests.get(f"{BASE_URL}/api/settings")
```

Ruff (0.15.12)

[error] Probable use of `requests` call without timeout

(S113)